src/styles/buy.css

```diff
@@ -0,0 +1,40 @@
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 24px ;
+}
+
+.price {
+  font-weight: 400px;
+  color: rgb(97, 96, 112);
+}
+
+.card__price-dolar {
+  font-weight: 700;
+  font-size: 16px;
+  color: rgb(6, 11, 53);
+}
+
+.card__buy {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  box-sizing: border-box;
+  background-color: #00acdc;
+  color: rgb(255, 255, 255);
+  height: 40px;
+  border-radius: 5px;
+  font-weight: 700px;
+  line-height: 16px;
+  font-size: 14px;
+  text-transform: uppercase;
+  text-decoration: none;
+  margin-top: 16px;
+}
+
```

Comment:
We should have independent CSS files for each BEM block. Here you have buy.css but when you name it, it looks like it is an element of the card as the name suggests. You might re-consider the naming of the classes according to BEM conventions.